Source/Core/VideoBackends/Metal/MTLPipeline.mm

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "VideoBackends/Metal/MTLPipeline.h"
+
+#include "Common/MsgHandler.h"
+
+static void MarkAsUsed(u32* list, u32 start, u32 length)
+{
+  for (u32 i = start; i < start + length; ++i)
+    *list |= 1 << i;
+}
+
+static void GetArguments(NSArray<MTLArgument*>* arguments, u32* textures, u32* samplers,
+                         u32* buffers)
+{
+  for (MTLArgument* argument in arguments)
+  {
+    u32 idx = [argument index];
+    u32 length = [argument arrayLength];
+    if (idx + length > 32)
+    {
+      PanicAlertFmt("Making a MTLPipeline with high argument index {:d}..<{:d} for {:s}",  //
```

Comment:
Stray `//`?

Reply:
Structural `//`.  Prevents `idx` from falling upwards onto this line during formatting